resources/user-agents/email-clients/pocomail/pocomail.json

```diff
@@ -0,0 +1,33 @@
+{
+  "division": "PocoMail",
+  "sortIndex": 1670,
+  "lite": false,
+  "standard": true,
+  "userAgents": [
+    {
+      "userAgent": "PocoMail",
+      "properties": {
```

Comment:
It should be safe to set the platform and device on this one to `Windows`/`Windows Desktop`, as pocomail was only ever released for Windows: http://www.pocomail.com/home/index.php?option=com_content&task=view&id=5&Itemid=28

Also on that page is a mention of the `3.4` version, if you care to add that.